openmicroscopy/openmicroscopy
# Rnd settings

This PR fixes a synchronization issue reported on the following card: https://trello.com/c/4mDNygGv/38-follow-up

To test:
 1. Log as user-6. Select the read-write group
 * Add user-5 to the display. 
 * Select the first image in the ```archived dv``` dataset under user-5
 * Go to preview tab
 * The image has been rnd settings for both user-5 and user-6
 * Select user-5 settings.
 * Click ```Launch Full Viewer```
 * Display the rendering controls (click on the left icon in the toolbar to display it)
 * Check that the thumbnail under user settings corresponding to user-5 is selected in the main viewer (it was user-6 w/o this PR)
 * Close the viewer w/o saving the settings.
 * Check that the settings are reset in the preview and the original thumbnail under User settings is selected i.e. user-6

Repeat the same test but with the following variation on step 7
  * one test with ```Click on Full Viewer``` in the general pane.
 * one test with ```Right click on the selected image in the tree```
 * one test with ```Right click on the selected image in the central pane```

cc @dominikl
